woocommerce/woocommerce
# Add 27 Egypt regions #26842

### All Submissions:


### Changes proposed in this Pull Request:

Closes #26842.

### How to test the changes in this Pull Request:

1. In `WooCommerce / Settings / Shipping / Shipping zones` if you search for Egypt, 26 Egypt regions should come up
2. On `Checkout page` once Egypt is selected as a Billing/Shipping country, new Egypt regions should be available in State / County field.

### Other information:


### Changelog entry

Enhancement - Add 27 Egypt regions.
